server/build-osx.sh

```diff
@@ -270,7 +270,7 @@ _build_server_osx() {
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libtiff*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libtiff"
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libiconv*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libiconv"
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libpng16*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libpng16"
-    #ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libexpat*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libexpat"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libexpat*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libexpat"
```

Comment:
what is expat needed now? does the license file include it's license?

Reply:
While testing the installer, SB was not launching if i dont bundle expat, it was throwing error for expat. So added it back to the script. Yes license file has its license.